src/main/java/com/pinterest/secor/util/orc/VectorColumnFiller.java

```diff
@@ -123,7 +123,7 @@ public void convert(JsonElement value, ColumnVector vect, int row) {
                             new Timestamp(value.getAsLong()));
                 } else {
                     if (!back.isBackOff()) {
-                        LOG.warn("Timestamp format is neither String nor Number");
+                        LOG.warn(String.format("Timestamp format is neither String nor Number: %s", value.toString()));
```

Comment:
I thought usually you will log using this format:

LOG.warn("Timestamp is neither string nor number: {}", value);

Reply:
@HenryCaiHaiying you are correct. I should follow that.